docs/modules/ROOT/pages/how-tos/install/gcp.adoc

```diff
@@ -37,6 +37,7 @@ It also will become a subdomain to the base domain.
 `<service account key>`:: See <<how-tos/gcp/project.adoc#service-account-key, Service Account Key>>
 `<region>`:: The GCP region to place the cluster. Default to `europe-west6`
 
+.Install cluster
```

Comment:
This is a title for the next paragraph which is a list item. Plus: why is this within that PR? It is not related.

Reply:
Currently the instructions are under the "Required input" paragraph: https://openshift.docs.vshn.ch/oc4/how-tos/install/gcp.html

Reply:
I see what the issues is. When rendered, one can not easily understand the scoping of the block titles. To solve this, we need to replace the the block titles with real section titles. I would prefer to do so within a PR on its own.

Reply:
See https://github.com/appuio/openshift4-docs/pull/22.